README.md

```diff
@@ -114,15 +168,18 @@ If changes to the database schema are made then the docker image for the databas
 
 ### Active Maintainers
 
-- **Selwyn Preston**, Lead Developer at London Borough of Hackney (selwyn.preston@hackney.gov.uk)
-- **Mirela Georgieva**, Lead Developer at London Borough of Hackney (mirela.georgieva@hackney.gov.uk)
-- **Matt Keyworth**, Lead Developer at London Borough of Hackney (matthew.keyworth@hackney.gov.uk)
+- **Faisal Gazi**, Lead Software Engineer at London Borough of Hackney (faisal.gazi@hackney.gov.uk)
+- **Humairaa Mulla**, Senior Software Engineer at London Borough of Hackney (humairaa.mulla@hackney.gov.uk)
+- **George Schena**, Senior Software Engineer at London Borough of Hackney (george.schena@hackney.gov.uk)
 
 ### Other Contacts
 
-- **Rashmi Shetty**, Product Owner at London Borough of Hackney (rashmi.shetty@hackney.gov.uk)
+- **Selwyn Preston**, Head of Engineering at London Borough of Hackney (selwyn.preston@hackney.gov.uk)
 
 [docker-download]: https://www.docker.com/products/docker-desktop
+[docker-compose-download]: https://docs.docker.com/compose/install/
 [universal-housing-simulator]: https://github.com/LBHackney-IT/lbh-universal-housing-simulator
 [made-tech]: https://madetech.com/
```

Comment:
Do we need made tech here?

Reply:
Removed the link in https://github.com/LBHackney-IT/housing-finance-interim-api/pull/114/commits/945916eac09932b12469c4a16f5f23fc5a40090d